apps/web/src/app/tools/send/send.component.html

```
@@ -79,29 +77,38 @@ <h1>
             </ng-container>
           </small>
```

We need to do something about this, but might be out of scope for now. We're essentially abusing the `appApiAction` to catch errors and to show a spinner. The spinner could be handled by just checking if `actionPromise` is null, but that would silently swallow errors.

---

I will open a tech-debt task for this